iron-image.html

```diff
@@ -224,70 +262,91 @@
           type: Number,
           value: null
         },
-
-        _placeholderBackgroundUrl: {
-          type: String,
-          computed: '_computePlaceholderBackgroundUrl(preload,placeholder)',
-          observer: '_placeholderBackgroundUrlChanged'
-        },
-
-        requiresPreload: {
```

Comment:
Also, if we consider undocumented properties as having undefined behavior, this PR is *technically* backwards compatible.. :P